.github/workflows/main.yml

```diff
@@ -90,15 +92,11 @@ jobs:
         source ${{github.workspace}}/.bashrc
         cd ${{github.workspace}}/ros2
         echo "================= ROSDEP UPDATE ====================="
-        # temp fix for rolling sources.. TODO: track when we can remove the two commands below
-        # see https://discourse.ros.org/t/psa-rolling-ci-or-docker-build-fix-from-rosdep-errors-in-24-04-transition/36902
-        sudo sed -i "s|ros\/rosdistro\/master|ros\/rosdistro\/rolling\/2024-02-28|" /etc/ros/rosdep/sources.list.d/20-default.list
-        export ROSDISTRO_INDEX_URL=https://raw.githubusercontent.com/ros/rosdistro/rolling/2024-02-28/index-v4.yaml
         rosdep update --rosdistro ${{ matrix.ros_distro }} --include-eol-distros
         echo "================= ROSDEP INSTALL ===================="
         rosdep install -i --reinstall --from-path src --rosdistro ${{ matrix.ros_distro }} --skip-keys=librealsense2 -y
         echo "================== COLCON BUILD ======================"
-        colcon build  --cmake-args '-DBUILD_TOOLS=ON'
+        colcon build  --cmake-args '-DBUILD_TOOLS=ON' --no-warn-unused-cli
```

Comment:
nice catch :)
Just an explanation to the record:
BUILD_TOOLS is relevant only for realsense2_camera package, so a warnings on unused cmd line variable were printed for realsense2_camera_msgs and realsense2_description.
now with `--no-warn-unused-cli`, there will be no warnings.
ref: https://cmake.org/cmake/help/v3.0/manual/cmake.1.html
`--no-warn-unused-cli`
Don’t warn about command line options.
Don’t find variables that are declared on the command line, but not used.

Reply:
Nice!
I suggest adding a short sentence as a comment above this line.
Same for python venv usage